pkg/tsdb/loki/frame_test.go

```diff
@@ -112,6 +120,13 @@ func TestAdjustFrame(t *testing.T) {
 			return frame
 		}
 
+		verifyFrame := func(frame *data.Frame, expectedFrameName string) {
+			require.Equal(t, frame.Name, expectedFrameName)
+			require.Equal(t, frame.Meta.ExecutedQueryString, "Expr: up(ALERTS)\nStep: 42s")
+			require.Equal(t, frame.Fields[0].Config.Interval, float64(42000))
+			require.Equal(t, frame.Fields[1].Config.DisplayNameFromDS, "legend Application")
```

Comment:
The "official" syntax is first the expected argument, then the actual. I think that's mixed up here, right?

Reply:
good catch. will fix it in a separate PR, i hope that's ok.